gravityflow/gravityflow
# Skip setting the gravityflow_is_fulfilled meta when there are no steps to process

Updates `Gravity_Flow::maybe_delay_workflow()` to abort early if the form does not have at least one active step to process.

**Testing Instructions**
- Submit a form without any steps
- Check the db and find that gravityflow_is_fulfilled is set for the entry
- Switch to this branch
- Repeat test and find gravityflow_is_fulfilled is not set
- Add an approval step and set it to inactive
- Submit the form and find gravityflow_is_fulfilled is not set
- Set the step to active
- Submit the form and find gravityflow_is_fulfilled is set
